src/virtualenv/seed/embed/wheels/__init__.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import absolute_import, unicode_literals
+
+BUNDLE_SUPPORT = {
+    "3.9": {"pip": "pip-19.3.1-py2.py3-none-any.whl", "setuptools": "setuptools-42.0.2-py2.py3-none-any.whl"},
```

Comment:
this seems to imply that newer versions will require a  change in `virtualenv` -- should it (does it already?) default to the latest?

(I had mostly fixed this in `tox` to not need updates every new version, pyflakes is growing some code for this too)

Reply:
newer versions automatically use MAX (see few lines below); we still check for wheel compatibility so it's alright